modules/geoipbackend/geoipbackend.cc

```diff
@@ -210,6 +244,23 @@ void GeoIPBackend::initialize() {
         nmt.insert(Netmask("::/0")).second.swap(value);
       }
 
+      // Allow per domain override of mapping_lookup_formats and custom_mapping.
+      // If not defined, the global values will be used.
+      if (YAML::Node formats = domain["mapping_lookup_formats"]) {
+        vector<string> mlf = formats.as<vector<string>>();
```

Comment:
Similarly, you could use `mapping` or `mapping_lookup_formats`

Reply:
It was for `mlf` right ? Done